meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -368,6 +372,24 @@ python() {
 # Helpers and internal variables.
 ################################################################################
 
+# @return the device portion of the uboot environment path (IE mmcblk0boot1, mmcblk0boot0 or mmcblk0)
+def mender_get_uboot_env_mmc_linux_device(d):
+    # Convert u-boot partition numbers to linux device references
+    # See [CONFIG_SYS_MMC_ENV_PART](https://github.com/u-boot/u-boot/blob/u-boot-2016.09.y/README#L4390) and
+    # [linux kernel mmc-dev-parts documentation](https://www.kernel.org/doc/Documentation/mmc/mmc-dev-parts.txt)
+    storage_device=d.getVar("MENDER_STORAGE_DEVICE",True)
+    env_part=d.getVar("MENDER_UBOOT_CONFIG_SYS_MMC_ENV_PART",True)
+    env_part_append=""
+    if env_part == "1":
+        env_part_append="boot0"
+    elif env_part == "2":
+        env_part_append="boot1"
```

Comment:
Any reason this can't be made generic? It is essentially `"boot%d" % (int(env_part) - 1)`, right? Unless `env_part == 0`.

Reply:
This is a better way to do it.  Addressed in [force push](https://github.com/mendersoftware/meta-mender/compare/f92f8114ca70418350489e978f3ae198f578d045..f65ae24861b63bcd85e22fb9fa08236339d9a801).